Kitodo/rulesets/ruleset.xsd

```diff
@@ -347,6 +363,7 @@
         <xs:attribute type="xs:boolean" name="editable" default="true"/>
         <xs:attribute type="xs:boolean" name="excluded" default="false"/>
         <xs:attribute type="xs:boolean" name="multiline" default="false"/>
+        <xs:attribute type="ruleset:Reimport" name="reimport" default="replace"/>
```

Comment:
So far it has been common that we always use the default which reflects the previous functionality. This is, so that after an update, the administrators have to make as few adjustments as possible to keep the system working as before. And the previous behavior was, that the metadata was replaced. And, I'd be `add` for the default behavior, by the way.

Reply:
Can not be the default value somehow configurable and not hard-coded?

Reply:
one question would be, wether this setting should be configurable
- ruleset-wide?
- project-wide?
- client-wide?
- system-wide?

I would probably also prefer to define it in the ruleset because it is a user decision and not an administartor decision in my opinion.